lldb/test/API/tools/lldb-server/memory-tagging/TestGdbRemoteMemoryTagging.py

```diff
@@ -84,7 +84,7 @@ def prep_memtags_test(self):
     @skipUnlessArch("aarch64")
     @skipUnlessPlatform(["linux"])
     @skipUnlessAArch64MTELinuxCompiler
-    def test_qMemTags_packets(self):
+    def test_qMemTags_packets_A(self):
```

Comment:
I've seen this with a packet test before, the GDB protocol likes to use first character lower case for read and upper case for write.

https://sourceware.org/gdb/current/onlinedocs/gdb.html/General-Query-Packets.html

So yes let's change the name but how about this instead:
```
test_tag_read_qMemTags_packets
test_tag_write_QMemTags_packets
```

Then it's clearer why there are 2 tests.

Reply:
Done.